Luminary099/PINBALL_GAME_BUTTONS_AND_LIGHTS.agc

```diff
@@ -185,23 +185,23 @@
 #			FOR DSPMMJB WITH PRIO 30000 AND RETURNS TO CALLER.)
 #
 #	BLANKSUB	ROUTINE BY WHICH AN INTERNAL PROGRAM MAY BLANK ANY
-#			COMBINATION OF THE DISPLAY REGISTERS R1, R2, R3.
+#			COMBINATION OF THE DISPLAY REGISTERS R1. R2. R3.
```

Comment:
These are commas in the original scans

![image](https://github.com/user-attachments/assets/f552003d-6251-43af-bdfe-d15ee8a91866)